pkg/operator/api/api.go

```diff
@@ -57,19 +57,17 @@ var clusterAccessRules = []v1beta1.PolicyRule{
 
 type Cluster struct {
 	context   *clusterd.Context
-	placement k8sutil.Placement
-	Name      string
 	Namespace string
+	placement k8sutil.Placement
 	Version   string
 	Replicas  int32
 }
 
-func New(context *clusterd.Context, name, namespace, version string, placement k8sutil.Placement) *Cluster {
+func New(context *clusterd.Context, namespace, version string, placement k8sutil.Placement) *Cluster {
 	return &Cluster{
 		context:   context,
-		placement: placement,
-		Name:      name,
```

Comment:
I actually like name. This is to identify the cluster, kinda like an ID. Even if we are only supporting one cluster per namespace, I dont think namespace should be used as identifiers for the clusters. This would be more important and more apparent once we build the cluster API. So that the route could look like `/v1/clusters/{cluster-name}`